mmdet/evaluation/metrics/crowdhuman_metric.py

```diff
@@ -521,11 +503,9 @@ def get_ignores(self, dt_boxes, gt_boxes):
 
 class Image(object):
     """Data structure for evaluation of CrowdHuman.
-
```

Comment:
Why do we need clean thess empty lines? Will it affect the rendered doc in readthedocs?

Reply:
It was an accident. I have already modified.